pkg/proc/variables.go

```diff
@@ -214,6 +214,146 @@ type G struct {
 	labels *map[string]string // G's pprof labels, computed on demand in Labels() method
 }
 
+// GoroutinesInfo searches for goroutines starting at index 'start', and
```

Comment:
This two functions don't really have anything to do with the rest of this file, they should stay at their original location in proc.go.

Also I'd like to keep the name of the receiver as dbp to preserve git history.